src/BusinessLayer/CommunicationService.cpp

```diff
@@ -25,14 +26,30 @@ void CommunicationService::setPeripheralSerialPort()
     outputPeripheral_ = new SerialPortPeripheral(*outputDevice_);
 }
 
-void CommunicationService::sendData(const unsigned char* packet, int packetLength)
+void CommunicationService::setInternetConnection()
 {
-    outputPeripheral_->sendData(packet, packetLength);
+    //function stub
```

Comment:
General practice is to leave a TODO. This way you can search a codebase for just TODOs.